bitnami/matomo/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: matomo
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/matomo
-version: 7.0.4
+version: 7.0.5
```

Because this PR adds a new feature, version should be bumped a minor instead of a patch:

```suggestion
version: 7.1.0
```